TicTacToe.py

```diff
@@ -4,15 +4,12 @@ def __init__(self,lst):
         self.lst=lst

```

Comment:
@JaspalSingh1759 , Please Add some Comment on What Function is Doing What. It Will Help Other 
Thanks